src/auth/auth.service.ts

```diff
@@ -0,0 +1,52 @@
+import { ForbiddenException, Injectable } from '@nestjs/common'
+
+import PrismaService from 'src/prisma/prisma.service'
+import { JwtService } from '@nestjs/jwt'
+import { ConfigService } from '@nestjs/config'
+import * as argon from 'argon2'
+import UpdateAuthDto from './dto/signinAuth.dto'
+
+@Injectable()
+export default class AuthService {
+  constructor(
+    private prisma: PrismaService,
+    private jwt: JwtService,
+    private config: ConfigService,
+  ) {}
+
+  async signin(dto: UpdateAuthDto) {
+    const user = await this.prisma.user.findUnique({
```

Comment:
Add exception handling to service methods.
If possible, add implementation of global exception filter.